video/out/wayland_common.c

```diff
@@ -56,6 +56,9 @@ const struct m_sub_options wayland_conf = {
     },
 };
 
+#define POINTER_EDGE_PIXELS 5
+#define TOUCH_EDGE_PIXELS 64
```

Comment:
Should these really be defines? Maybe it makes more sense to expose these as user configurable options?

Reply:
Would you want one option for each or just one option?